### PR TITLE
fix PUD-1103 (pci.storage.databases): Fix the polling of users status

### PR DESCRIPTION
**Pull request by jperchoc**
PUD-1103

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #PUD-1103
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Fix the polling of users status on databases tab "users", use specific database tracking

### :flags: Translations

- [x] TRANS-40387

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/users/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "pci_databases_users_title": "Users & Roles",
+  "pci_databases_users_username_label": "Nom d'utilisateur",
+  "pci_databases_users_description_label": "Description",
+  "pci_databases_users_role_label": "Rôles",
+  "pci_databases_users_createdAt_label": "Date de création",
+  "pci_databases_users_status_label": "Status",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/users/translations/Messages_fr_FR.json:7]:
```suggestion
  "pci_databases_users_status_label": "Statut",
```

Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/users/translations/Messages_fr_FR.json]: Corrected here: https://github.com/ovh/manager/pull/6090/commits/548d9e809657d6083ad9d5b22cd0a05986765743
Thank you !

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                ></span>
+            </oui-action-menu-item>
+        </oui-action-menu>
+
+        <oui-datagrid-topbar>
+            <oui-button
+                data-disabled="$ctrl.database.isProcessing()"
+                data-on-click="$ctrl.trackAndAddUser()"
+            >
+                <i class="oui-icon oui-icon-add pr-1" aria-hidden="true"></i>
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.html:111]:
```suggestion
                <span class="oui-icon oui-icon-add pr-1" aria-hidden="true"></span>
```

**Review by JeremyDec — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.html (hunk 2 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.html]: Thanks, changed here: https://github.com/ovh/manager/pull/6090/commits/c05379f707d6d1fdb276a6ceea7bc16f736e7b75

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/users/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "pci_databases_users_status_error": "En erreur",
+  "pci_databases_users_status_error_inconsistent_spec": "En erreur",
+  "pci_databases_users_status_pending": "En attente",
+
+  "pci_databases_users_modify_password_action_label": "Régénérer le mot de passe",
+  "pci_databases_users_show_key_action_label": "Afficher la clé d'accès",
+  "pci_databases_users_show_cert_action_label": "Afficher le certificat",
+  "pci_databases_users_show_user_informations_action_label": "Afficher les informations",
+
+  "pci_databases_users_password_message_infos": "Le nouveau mot de passe de l'utilisateur {{ user }} est <code>{{ password }}</code>.",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/users/translations/Messages_fr_FR.json:23]: **question:** Having trouble to find where `pci_databases_users_password_message_*`  translations are used. Are they really needed?

**question:** Would it be possible to remove `<code>` tag from translation by using template concatenation?

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.routing.js [hunk 4]
```diff
@@ -1,145 +1,60 @@
...
       resolve: {
         breadcrumb: /* @ngInject */ ($translate) =>
-          $translate.instant('pci_projects_project_users_title'),
-        users: /* @ngInject */ (DatabaseService, database, projectId) =>
-          DatabaseService.getUsers(
+          $translate.instant('pci_databases_users_title'),
+        users: /* @ngInject */ (DatabaseService, database, projectId) => {
+          // Poll the database status
+          if (database.isProcessing()) {
+            DatabaseService.pollDatabaseStatus(
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.routing.js:20]: **question:** Does the Polling is killed once user move to another state? (Used to be done on `$onDestroy` hook).

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/users/translations/Messages_fr_FR.json (hunk 3 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/users/translations/Messages_fr_FR.json]: Those translations are indeed not used anymore, I have removed them, thanks !

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.routing.js (hunk 4 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.routing.js:20]: Good catch, I forgot to stop the polling, I've added it here: https://github.com/ovh/manager/pull/6090/commits/ea60c8f86c638103f84054aff088896cc6192f2b

**Review by antleblanc — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]
suggestion: `aria-label` can be added for `oui-action-menu-item` and `oui-button`

**Review by JeremyDec — APPROVED** [review]

**Review by radireddy — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**